test/test_rules.py

```diff
@@ -1,16 +1,15 @@
 from opimodel import rules, colors
 
 
-def test_empty_RulesNode(widget, get_renderer):
+def test_empty_RulesNode_does_not_create_rules_tag(widget, get_renderer):
```

Comment:
This test needs updating now the rules list is not API, or a second test adding to check "no rules node rendered when no rules added"